### PR TITLE
update templates to address things

### PR DESCRIPTION
**Pull request by AbnormalPoof**
Bug and compiling issue templates: Added disclaimer saying to type in the body and not the title

Bug: fix grammar error and changed title template
PR enhancement template: added disclaimer saying not to add thousands of commits from other engines

### CODE REVIEW

**Comment by prokube** [comment]
> Bug and compiling issue templates: Added disclaimer saying to type in the body and not the title
> PR enhancement template: added disclaimer saying not to add thousands of commits from other engines

you can't win an argument against an idiot

File: .github/PULL_REQUEST_TEMPLATE/enhancement.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ title: 'Enhancement: '
 labels: enhancement
 ---
 #### Please check for duplicates or similar PRs before creating this issue.
+#### Do NOT make PRs with thousands of commits from other engines, as they will likely never be official.
```
Comment (Cheemsandfriends) [inline .github/PULL_REQUEST_TEMPLATE/enhancement.md:8]: This is really needed

**Review by Cheemsandfriends — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/enhancement.md (hunk 1 above)
Comment (AbnormalPoof) [reply .github/PULL_REQUEST_TEMPLATE/enhancement.md:8]: about time someone addressed the issue

Comment (Steviegt6) [reply .github/PULL_REQUEST_TEMPLATE/enhancement.md:8]: > `as they will likely never be official`
Unnecessary bloat.

**Review by Steviegt6 — CHANGES_REQUESTED** [review]

**Comment by AbnormalPoof** [comment]
@Steviegt6 better?

**Comment by Steviegt6** [comment]
> @Steviegt6 better?

Yeah. Doubt anyone making those kinds of PRs would understand how cherry-picking works but it's a definite improvement. Thanks!

**Review by Steviegt6 — APPROVED** [review]

**Comment by AbnormalPoof** [comment]
No problem lol

**Pull request closed without merging**